scripts/test-node-api.js

```diff
@@ -0,0 +1,7 @@
+const assert = require("assert")
+assert(require.resolve('@lingui/cli/api'));
+assert(require.resolve('@lingui/cli/api/extractors/typescript.js'));
+assert(require.resolve('@lingui/cli/api/extractors/babel.js'));
+
+assert(require.resolve('@lingui/core'));
+assert(require.resolve('@lingui/core/compile'));
```

Comment:
kinda quick and dirty smoke test that at least all entries we consider public are really exposed.